collect_app/src/main/res/values/strings.xml

```diff
@@ -701,4 +701,5 @@
     <string name="copying_media_files_failed">Copying media files failed</string>
     <string name="qr_code_not_found">QR Code not found in the selected image</string>
     <string name="not_granted_permission">The activity you are trying to start requires a permission which is not granted.</string>
+    <string name="mapbox_unsupported_warning"><![CDATA[Mapbox is not supported on this device (%s).  Please choose another mapping SDK in General Settings > User interface.]]></string>
```

Comment:
Would you consider escaping `&gt;` instead of using a CDATA block? I think this would confuse translators more.